src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js

```diff
@@ -376,7 +385,7 @@ const InstructorAddCourseForm = props => {
               type="submit"
               htmlType="submit"
               className="submitButton"
-              onClick={success}
```

Comment:
Are we no longer using the success function on line 108? If so, should delete it?

Reply:
thank you I missed that, moved the success modal into the submit function